### ISSUE TITLE
Tree-View on proofs

### ISSUE THREAD

**Issue by gleachkr** [gleachkr/Carnap-Old#16]
We'd like to allow for multiple views on natural deduction derivations, including a tree view.

**Comment by JakeEhrlich** [comment]
Are we using MathJax already?

**Comment by gleachkr** [comment]
Nope, just unicode. I'll check if MathJax supports drawing trees.
